### DISCUSSION TITLE
Is Jsstore treeshakable?

### DISCUSSION THREAD

**Discussion by Septias** [ujjwalguptaofficial/JsStore#324] [Q&A] [answered]
Question in title

**Comment by ujjwalguptaofficial** [comment, marked as answer]
No - JsStore has been created thinking about using it in worker and that's why all of the logic is inside a worker file. That's why it can not be treeshakable - it handles all the web worker communication and based on the communication it executes code which are like everythings are tightly packed. 

I hope this clearifies.

**Comment by Septias** [comment]
Okay, thanks
